src/main/java/com/itechart/javalab/library/dao/BookDao.java

```diff
@@ -11,6 +11,5 @@
 
     Optional<List<Book>> getBooks(Paginator paginator, BookFilter bookFilter);
     Optional<Integer> getNumberOfBooksRecords(BookFilter bookFilter);
-
-
+    Optional<List<Book>> findBooksByParameters(BookFilter bookFilter);
```

Comment:
why `getBook`s and `findBooksByParameters` takes the same parameters, but only `findBooksByParameters` can truly does the filter job